### ISSUE TITLE
Rename Mentor/Mentoree to MentorProfile/MentoreeProfile

### ISSUE THREAD

**Issue by lbighetti** [lbighetti/hades#37]
I think this would be more explicit what the idea is, as the intended use of these resources are as a "role" of user. Do you think that makes sense ?

**Comment by ccarvalho-eng** [comment]
Yes, makes sense. Not necessarily needed but it would make things much more clear to new devs.
